doc/conf.py

```diff
@@ -62,6 +62,7 @@
     "support": "contact.html",
     "user_guide/ide-integration": "installation.html",
     "user_guide/message-control": "user_guide/messages/message_control.html",
+    "additional_commands/index": "index.html",
```

Comment:
This redirect doesn't work 😓

Reply:
I don't understand why redirecting in sphinx is so freaking hard 😄

Reply:
https://github.com/sphinx-contrib/redirects seems serious and 'simple' how about that ?

Reply:
Yeah that's the one I was originally looking at originally. It's not "official" but the organization gives me some hope.
It doesn't support pattern matching, which is something we'll need to keep using `RTD` for I guess. (You might want to subscribe to https://github.com/readthedocs/readthedocs.org/issues/4221 which makes it so contributors can also help make redirects)

On the other hand, the maintainer of `reredirects` is quite responsive (I have actually gotten two maintenance PRs merged already). So if we can specify what is currently missing we can also ask them to fix it?

Reply:
For future reference, we're trying to get https://pylint--6641.org.readthedocs.build/en/6641/additional_commands/index.html to work.

Reply:
```suggestion
    "additional_commands/index": "../index.html",
```

This works! 🎉

Reply:
Link for test: https://pylint--6641.org.readthedocs.build/en/6641/additional_commands/index.html (right now it's an infinite redirect)

Reply:
It is also good to know why. Apparently these redirects are relative. So it kept redirecting from `additional_commands/index.html` to ... `additional_commands/index.html` 😅

Reply:
Yeah this is actually the bug that made it click for me.